.changeset/spicy-mails-sort.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+docs: improve `cookbook/transaction-request`
```

Comment:
I think this deserves to be included in release notes.

Reply:
@danielbate, I was unaware that we could include changes that did not affect any package in the release notes. How can I achieve this?

Reply:
Just by adding release notes in PR desc I believe? 🤔